steps/id-and-residency/other-support.ts

```diff
@@ -0,0 +1,133 @@
+import { Heading, HeadingLevels } from "lbh-frontend-react";
+import {
+  ComponentDatabaseMap,
+  ComponentWrapper,
+  DynamicComponent,
+  StaticComponent
+} from "remultiform/component-wrapper";
+import { makeSubmit } from "../../components/makeSubmit";
+import { TextInput } from "../../components/TextInput";
+import keyFromSlug from "../../helpers/keyFromSlug";
+import nextSlugWithId from "../../helpers/nextSlugWithId";
+import ProcessStepDefinition from "../../helpers/ProcessStepDefinition";
+import ResidentDatabaseSchema from "../../storage/ResidentDatabaseSchema";
+import Storage from "../../storage/Storage";
+import PageSlugs from "../PageSlugs";
+import PageTitles from "../PageTitles";
+
+const questions = {
+  "other-support-details": "Other support details"
+};
+
+const step: ProcessStepDefinition<ResidentDatabaseSchema, "otherSupport"> = {
+  title: PageTitles.OtherSupport,
+  heading: "Other Support",
+  context: Storage.ResidentContext,
+  review: {
+    rows: [
+      {
+        label: questions["other-support-details"],
+        values: {
+          "other-support-full-name": {
+            renderValue(name: string): React.ReactNode {
+              return name;
+            }
+          },
+          "other-support-role": {
+            renderValue(role: string): React.ReactNode {
+              return role;
+            }
+          },
+          "other-support-phone-number": {
+            renderValue(phoneNumber: string): React.ReactNode {
+              return phoneNumber;
+            }
+          }
+        }
+      }
+    ]
+  },
+  step: {
+    slug: PageSlugs.OtherSupport,
+    nextSlug: nextSlugWithId(PageSlugs.Verify),
```

Comment:
ironically, you need to not wrap `verify` here, as we don't need to pass each individual tenant's `id` here, so it's just `PageSlugs.Verify`, which is why you were getting the 404 before